src/WinUIEx/MonitorInfo.cs

```diff
@@ -69,8 +71,13 @@ internal unsafe MonitorInfo(HMONITOR monitor, RECT* rect)
         /// </remarks>
         public Rect RectWork { get; }
 
+        /// <summary>
+        /// Gets if the monitor is the the primary display monitor.
+        /// </summary>
+        public bool IsPrimary { get; }
```

Comment:
I wonder if it would be better to move line 50 to here:
`public bool IsPrimary => _monitor == PInvoke.MonitorFromWindow(new(IntPtr.Zero), MONITOR_FROM_FLAGS.MONITOR_DEFAULTTOPRIMARY);`
That ensures it is up to date, in case of changes, and doesn't require the extra interop call if you're never actually checking this

Reply:
Oh, it makes sense.